Fix colored strings to restore original terminal color

mpluders opened the pull request:
A potential fix for #163. It simply appends a white Ansi color code when printing colored strings. This successfully restores the original terminal color.

I'm not sure how you would create a unit test for something like this, but would love to hear any suggestions.

rastersize commented on live/sources/String+PrintWithColor.swift:
```diff
@@ -29,6 +29,7 @@ extension String {
     func print(withColor color: AnsiColor) {
-        Swift.print(color.rawValue + self)
+        /// Append a white color code to the string, to reset the console color
+        Swift.print(color.rawValue + self + AnsiColor.white.rawValue)
```
How about using the “reset” ANSI value instead? Setting it to white might cause issues for user’s that use a white background color and black foreground. The “reset” value instead just clears any custom formatting. Returning the text that comes afterwards to the terminals default colors.

The ansi value is:
`[0m`

So we’d add a enum value like:
```swift
case reset = "\u{001B}[0m"
```

JohnSundell replied:
Ah, that sounds a lot better! 👍 We actually don't need a case for it, we can just inline the string at the end of the `print` call.

mpluders commented:
Ahh that makes more sense, thanks for point that out @rastersize. I implemented the reset code in-line, and removed the superfluous case from AnsiColor.swift.

JohnSundell commented:
Awesome @mpluders! Looking great 👍
